exec/java-exec/src/test/java/org/apache/drill/exec/fn/interp/TestConstantFolding.java

```diff
@@ -117,6 +123,13 @@ public void createFiles(int smallFileLines, int bigFileLines) throws Exception{
 
   @Test
   public void testConstantFolding_allTypes() throws Exception {
+    new MockUp<DateTimeUtils>() {
```

Comment:
I meant to use the same method in all tests where Locale should be mocked. Please replace it.